packages/webdriverio/test/axe-webdriverio.spec.ts

```diff
@@ -110,7 +118,7 @@ describe('@axe-core/webdriverio', () => {
             browserName: 'chrome',
             'goog:chromeOptions': {
               args: ['--headless'],
-              binary: binaryPath
+              binary: binaryPath ?? (process.env.CHROME_TEST_PATH as string)
```

Comment:
I'm not entirely clear on the need for the Windows path here, but I'm deferring to it just in case.

Reply:
I think this should probably be dropped; it should be unnecessary now that `CHROME_TEST_PATH` is controlling the browser+driver version. Even on Windows, we want the tests to use the BDM-synced version of chrome that matches the version of chromedriver, so I don't think we should allow windows to use the globally-installed chrome version.